student.library/src/main/java/tds/student/sql/data/OpportunityStatusExtensions.java

```diff
@@ -13,12 +13,10 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import tds.exam.ExamStatusCode;
-
 public class OpportunityStatusExtensions
 {
   private static final Map<String, OpportunityStatusType> _statuses = new HashMap<> ();
-  private static final Map<String, OpportunityStatusType> examStatusToOpportunityStatus = new HashMap<>();
```

Comment:
These are static immutable lookup tables, not state?  So we could have two Student WARs running behind a load balancer?